.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -7,51 +7,64 @@ assignees: ''
 
 ---
 
-**Describe the bug**
-<!--A clear and concise description of what the bug is.-->
+<!-- 
+Explanation of applied labels can be found here: https://github.com/kirbydesign/designsystem/labels
 
-**To Reproduce**
-Steps to reproduce the behavior:
+The good issue: contains only one issue, is self-contained, is specific & unambigious, follows the template, has a good title, is easy to read and contains screenshots & -recordings
+-->
+
+[ ] I have written a [good issue](../CONTRIBUTING.md/#issue) 
```

Comment:
I would prefer if this was the first thing in the above comment instead, as I think it is better communicated as a guideline than a checkbox people will have to check every single time.